pkg/antctl/raw/traceflow/command.go

```diff
@@ -0,0 +1,148 @@
+// Copyright 2020 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package traceflow
+
+import (
+	"context"
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/spf13/cobra"
+	"gopkg.in/yaml.v2"
+	"k8s.io/apimachinery/pkg/util/wait"
+	"k8s.io/client-go/rest"
+
+	"github.com/vmware-tanzu/antrea/pkg/antctl/raw"
+	"github.com/vmware-tanzu/antrea/pkg/antctl/runtime"
+	"github.com/vmware-tanzu/antrea/pkg/apis/ops/v1alpha1"
+	"github.com/vmware-tanzu/antrea/pkg/ops"
+)
+
+var Command *cobra.Command
+var source string
+var destination string
+
+// Response is the response of antctl traceflow.
+type Response struct {
+	Name        string                  `json:"name" yaml:"name"`      // Traceflow name
+	Phase       v1alpha1.TraceflowPhase `json:"phase,omitempty"`       // Traceflow phase
+	Source      string                  `json:"source,omitempty"`      // Traceflow source, e.g. "default/pod0"
+	Destination string                  `json:"destination,omitempty"` // Traceflow destination, e.g. "default/pod1"
+	NodeResults []v1alpha1.NodeResult   `json:"results,omitempty"`     // Traceflow node results
+}
+
+func init() {
+	Command = &cobra.Command{
+		Use:     "traceflow",
+		Short:   "Start a Traceflows",
+		Long:    "Start a Traceflows from one pod to another.",
+		Aliases: []string{"tf", "traceflows"},
+		Example: ` Start a Traceflow from busybox0 to busybox1, both pods are in namespace default
+  $antctl traceflow -S busybox0 -D busybox1 
```

Comment:
We also have fields of "Source Port", "Destination Port" and "Protocol" on UI which allows uses to set. Do we need antctl to support this? @gran-vmv

Reply:
How about add these fields as optional?